config.cc

```diff
@@ -2221,7 +2256,19 @@ QString getPidFileName() THROW_SPEC( exError )
 
 QString getHistoryFileName() THROW_SPEC( exError )
 {
+  #ifdef Q_OS_LINUX
+    QDir home = getHomeDir();
+    if (home.dirName().endsWith(".goldendict"))
```

Comment:
Why `endsWith()`? Shouldn't `dirName()` return exactly `".goldendict"`?

Reply:
Maybe it would be better to check if *history* exists in the home dir, and if so, use it? Would accommodate users who want to keep *history* alongside the config files.

Reply:
One could achieve that as well setting XDG_DATA_HOME as XDG_CONFIG_HOME, but it's a good idea; changed in the latest version.